docs/pages/admin-guide.mdx

```diff
@@ -315,15 +293,13 @@ auth_service:
 ```
 
 <Admonition type="note">
-  SSO users can also register 2FA devices, but Teleport will not prompt them
-  for 2FA during login. Login 2FA for SSO users should be handled by the SSO
+  SSO users can also register 2FA devices, but Teleport will not prompt them for 2FA during login. Login 2FA for SSO users should be handled by the SSO
   provider.
 </Admonition>
 
-### Github OAuth 2.0 Connector
+### GitHub OAuth 2.0 connector
```

Comment:
Unrelated to the edit pass, we should pull this out into its own guide. It's hidden for OSS users compared to https://goteleport.com/docs/enterprise/sso/ssh-sso/ ( but maybe not in the enterprise page )

Reply:
Totally agree, I have an item on the Layout Itinerary for:

1. Access Controls > SSO general article
2. Access Controls > Single Sign-On (SSO) Section
3. Also, all of the Enterprise > Plugins will go under Access Controls

https://github.com/gravitational/teleport/blob/b6193fe722d76cd9f0084cfb875ef8be8590c645/docs/layout.md